recipes/machina/build.sh

```diff
@@ -0,0 +1,25 @@
+# Download and compile Gurobi 9.0.2
+wget https://packages.gurobi.com/9.0/gurobi9.0.2_linux64.tar.gz -O gurobi9.0.2_linux64.tar.gz
+tar xvzf gurobi9.0.2_linux64.tar.gz
+sed -i 's/C++/CPP/g' gurobi902/linux64/src/build/Makefile
+(cd gurobi902/linux64/src/build && make CPP=${CXX})
+(cd gurobi902/linux64/lib && ln -f -s ../src/build/libgurobi_c++.a libgurobi_c++.a)
```

Comment:
It might be better long-term to move the download and extraction of this to the meta.yaml:

```
source:
  - url: https://github.com/raphael-group/{{ name }}/archive/v{{ version }}.zip
    sha256: feb1efbedd32b8339f04699fade79fbc67bbd919df0a27f9a108f54c7531d208
  - folder: gurobi902
    url: https://packages.gurobi.com/9.0/gurobi9.0.2_linux64.tar.gz
    sha256: something
```

That would make the gurobi inclusion in the build more explicit. Can you give that a try?

Reply:
Thanks..going with this approach now!